src/Card/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React                from 'react';
+import PropTypes            from 'prop-types';
+
+import { buildClassName }   from '../utils';
+import styles               from './card.css';
+
+const Card = ( {
+    cssMap,
+    className,
+    children,
+    padding
+} ) => (
+    <div
+        className = { buildClassName( className, cssMap, {
+            paddingX : typeof padding === 'string' ? padding : padding[ 0 ],
+            paddingY : typeof padding === 'string' ? padding : padding[ 1 ]
```

Comment:
Super-minor thing: I think it would be slightly safer to check that the value is an array, rather than checking if it’s a string. So:
```
paddingX : Array.isArray( padding ) ? padding[ 0 ] : padding,
paddingY : Array.isArray( padding ) ? padding[ 1 ] : padding,
```

This way it wouldn’t throw an error if you passed in, say, an integer to the `padding` prop.